packages/opentelemetry-exporter-collector-grpc/src/CollectorExporterNodeBase.ts

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+import * as grpc from '@grpc/grpc-js'
```

Comment:
Non-blocking comment:
Just a general comment on minification, generally (depending on the final module format) it is better to just import the individual required exported elements rather than effectively importing every exported element. And looking at your changes in this file it *looks* like you could just import Metadata...

Reply:
The "MS" in MSNev stands for "Minification Snob" :)

Seriously though, this is good info and we should probably follow his advice here unless there's a reason not to.

Reply:
good catch, importing only what is needed should be the way to go here

Reply:
Good call, updated. 📦